index.html

```diff
@@ -4,20 +4,19 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>First mobile portfolio</title>
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&family=Roboto:wght@100;500&display=swap"
-      rel="stylesheet"
-    />
+    <title>First hand</title>
     <link rel="stylesheet" href="main.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
     <script
       src="https://kit.fontawesome.com/667d7efdcc.js"
       crossorigin="anonymous"
     ></script>
   </head>
   <body>
+    <main>
+
     <header class="header-mobile">
       <nav class="nav">
         <div class="logo"><a href="#"> I'm Hussein</a></div>
```

Comment:
- [OPTIONAL] I would implore you to align the cards just as it is aligned on figma so that it would look alike.
### Your work
![image](https://user-images.githubusercontent.com/86200356/227906097-6209008e-317e-4913-9ab4-5b36c92c69b0.png)
 ### Figma design
![image](https://user-images.githubusercontent.com/86200356/227906362-90287b73-ebf6-46f0-b1b1-a12a8b278dfb.png)

### Kindly apply this to all the cards as they are not appropriately aligned.